### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by tucker-weed**
Updates/adds commands

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -61,25 +61,34 @@ docker-compose up
 
 start service
 ```shell
-yarn service
+yarn service start
```
Comment (chewnoill) [inline readme.md:64]: `yarn service dev` is a little nicer for local development

File: readme.md [hunk 2]
```diff
@@ -61,25 +61,34 @@ docker-compose up
...
 ```
 
 start storybook
 ```shell
-yarn storybook
+yarn client dev
```
Comment (chewnoill) [inline readme.md:69]: my workflows here have moved more towards docker I think. What are you using?

**Review by chewnoill — APPROVED** [review]

File: readme.md (hunk 2 above)
Comment (tucker-weed) [reply readme.md:69]: @chewnoill been using the package.json script commands to run the app out of habit - I'll delete the startup instructions besides docker

Comment (chewnoill) [reply readme.md:69]: yarn scripts are probably fine, maybe less overhead? If we can get the nextjs middleware working for websockets it would be viable.

**Review by chewnoill — APPROVED** [review]

**Pull request merged**